src/main/java/zd/zero/waifu/motivator/plugin/player/WaifuSoundPlayerFactory.java

```diff
@@ -19,18 +19,20 @@ public static WaifuSoundPlayer createPlayer( Path fileName ) {
             .filter( f -> f.contains( "." ) )
             .map( f -> f.substring( f.lastIndexOf( '.' ) + 1 ) ).orElse( "" );
 
-        Optional<SupportedFile> supportedFile = SupportedFile.ofExtension( extension );
-        if ( supportedFile.isPresent() ) {
-            switch ( supportedFile.get() ) {
-                case CLIP:
-                    return DefaultWaifuSoundPlayer.ofFile( fileName );
-
-                case MP3:
-                    return Mp3WaifuSoundPlayer.ofFile( fileName );
-            }
-        }
-
-        return EmptyWaifuSoundPlayer.of( fileName );
+        return SupportedFile.ofExtension( extension )
```

Comment:
I could not resist changing this, when I see `isPresent` and `get` it can almost always be replaced with other `Optional` API operators. I can change it back, if it bothers you.